options/m_option.c

```diff
@@ -2608,13 +2608,41 @@ static char *pretty_print_rel_time(const m_option_t *opt, const void *val)
     return print_rel_time(opt, val);
 }
 
+static int rel_time_set(const m_option_t *opt, void *dst,
+                        struct mpv_node *src)
+{
+    struct m_rel_time *t = dst;
+    if (src->format == MPV_FORMAT_STRING) {
```

Comment:
Not needed. the generic code always calls parse for strings. See e.g. `int64_set`.

Reply:
Ok. `time_set()` had an explicit test for string, but it makes more sense that way - gone.